### PR TITLE
Idempotenize imports.

### PR DESCRIPTION
**Pull request by hvoecking**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Read a protobuf schema from the disk, parse it and resolve all imports",
   "main": "index.js",
   "dependencies": {
+    "lodash": "^4.8.2",
```
Comment (eckardt) [inline package.json]: I think we should avoid adding a new dependency here. What about hand-writing the necessary functions?

Comment (hvoecking) [inline package.json]: I agree that we shouldn't add a dependency for one trivial function. However, implementing and properly testing the logic of a bunch of different functions is worth considering introducing another dependency. Furthermore, _lodash_ is more or less a standard library. But, I'll see how much work it is if I rewrite the code a little.
:+1: https://github.com/hvoecking/resolve-protobuf-schema/commit/5f00d18

**Comment by eckardt** [comment]
LGTM

**Comment by hvoecking** [comment]
Will be merged by https://github.com/mbrtargeting/resolve-protobuf-schema/pull/3

**Pull request closed without merging**